src/Microsoft.AspNet.Mvc.Razor/ViewLocationExpanderContext.cs

```diff
@@ -18,10 +17,20 @@ public class ViewLocationExpanderContext
         /// <param name="actionContext">The <see cref="Mvc.ActionContext"/> for the current executing action.</param>
         /// <param name="viewName">The view name.</param>
         /// <param name="isPartial">Determines if the view being discovered is a partial.</param>
-        public ViewLocationExpanderContext([NotNull] ActionContext actionContext,
-                                           [NotNull] string viewName,
+        public ViewLocationExpanderContext(ActionContext actionContext,
+                                           string viewName,
                                            bool isPartial)
```

Comment:
Think this is the only fancy indentation visible in this PR. Great to see all the other cleanup.